.github/workflows/release.yml

```diff
@@ -22,6 +22,11 @@ jobs:
           java-version: 11
           distribution: 'zulu'
           cache: 'gradle'
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - uses: actions/setup-java@v2
```

Comment:
This is already done on L20?

Reply:
Yeah, looks like a copy-paste error, here's a pull request with a fix: https://github.com/spinnaker/rosco/pull/887